### PR TITLE
Added a template for PR

### PR DESCRIPTION
**Pull request by SvA1**
Please let me know if any fields for the template are missing.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+Please make sure that the PR fulfills these requirements
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I checked that there aren't other open Pull Requests for the same update/change.
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
+
```
Comment (JAdshead) [inline .github/pull_request_template.md:10]: - PR contains a single rule change
- Rule changes includes a comment to describe the reasoning behind the change

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how -->
+<!--- your change affects other areas of the code, etc. -->
+- Does your submission pass tests?
+- Have you successfully ran tests with your changes locally?
```
Comment (JAdshead) [inline .github/pull_request_template.md:20]: Don't think we need this
```suggestion
```

**Review by JAdshead — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please could you also reformat your commit message to follow our required commit format - https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary

**Comment by nellyk** [comment]
@SvA1  the PR looks good, could you please make the changes requested by @JAdshead

**Comment by SvA1** [comment]
> 
> 
> @SvA1 the PR looks good, could you please make the changes requested by @JAdshead

I'm sorry I can't edit this PR anymore, to add the changes I created a new PR, hope this works out

**Comment by nellyk** [comment]
closing this

**Pull request closed without merging**